dashboard/app/notifications_test.go

```diff
@@ -97,17 +97,39 @@ func TestEmailNotifBadFix(t *testing.T) {
 
 	expectReply := fmt.Sprintf(`This bug is marked as fixed by commit:
 some: commit title
-But I can't find it in any tested tree for more than 90 days.
+
+But I can't find it in the tested trees[1] for more than 90 days.
 Is it a correct commit? Please update it by replying:
+
 #syz fix: exact-commit-title
 
-Until then the bug is still considered open and new crashes with the same signature are ignored.
+Until then the bug is still considered open and new crashes with
+the same signature are ignored.
 
 Dashboard link: https://testapp.appspot.com/bug?extid=%s
+
+---
+[1] For the test2 namespace, I check commits in:
+
+1. branch10 branch of
+git://syzkaller.org
+
+2. branch20 branch of
+git://syzkaller.org
+
+3. extra-branch-1 branch of
+git://temp.org
+
+4. extra-branch-2 branch of
+git://temp.org
+
+The full list (#1..5) can be found at
```

Comment:
I did not immediately understood what "(#1..5)" means.
Maybe "The full list of 5 repositories (or trees?) can be found at"?